### PR TITLE
Show an icon that indicates whether a bug report activity has time lapse data.

### PR DESCRIPTION
**Pull request by sandonian**
Moving this pull request here from: https://github.com/sandonian/web-hv/pull/5, so that it can be merged when approved and so that email notifications work properly.

Below is the current UX:
image

### CODE REVIEW

**Comment by sandonian** [comment]
From the previous CL:

Sunny said:

Let's avoid the badging and change the primary icon itself.
Also current icon for activity list ('looks_one') looks like an error as if the time-lapse data is the real deal and everything else is legacy case.
Also the icon does not mean anything in the absence of a time-lapse data

Stefan replied:
1. I'll avoid the badge and just use the time lapse icon as the actual icon background.
2. Are you a) saying I should remove the looks_one icon completely, or b) only show it if there is at least one time lapse data present in a bug report activity list?

For reference, this is option a: 
image

and this is option b:
image

**Comment by sunnygoyal** [comment]
I meant, use completely different set of icons: which represent the entity on its own and not relative to other entities.
For example the icon "1" tries to indicate "1 frame of hierarchy". In this case the main property of the entity should be "hierarchy" and not "1 frame". So the icon should essentially indicate "hierarchy" and not that there is only one-of-it.

For the live-capture icon, 'multi-frame' is probably the correct property to prioritize. But the icon is too abstract right now for that.

Also use the xml/svg directly similar other icons instead of trying to find something from the limited set of material design icons.

**Comment by sandonian** [comment]
> I meant, use completely different set of icons: which represent the entity on its own and not relative to other entities. For example the icon "1" tries to indicate "1 frame of hierarchy". In this case the main property of the entity should be "hierarchy" and not "1 frame". So the icon should essentially indicate "hierarchy" and not that there is only one-of-it.
> 
> For the live-capture icon, 'multi-frame' is probably the correct property to prioritize. But the icon is too abstract right now for that.
> 
> Also use the xml/svg directly similar other icons instead of trying to find something from the limited set of material design icons.

That makes sense Sunny. I didn't understand why you liked the "graph"/"tree node" icons earlier in our gmail chat conversation about the icons, but your explanation above shines light on why you prefer a non-relational set of 2 icons.

What do you think of the following?
image

File: js/activity_list.js [hunk 1]
```diff
@@ -62,10 +62,14 @@ const activityListAction = function (initializer) {
             const entry = $("<div>").data("appInfo", l).appendTo(container).click(startHView).addClass("entry");
 
             if (list.hasIcons) {
```
Comment (sunnygoyal) [inline js/activity_list.js]: if we are always showing icons, can we get rid of this check?
Which case doesn't has icons?

Comment (sandonian) [reply js/activity_list.js]: Sounds good to me!

**Review by sunnygoyal — APPROVED** [review]

**Pull request merged**